app/models/auction_status.rb

```diff
@@ -19,6 +19,10 @@ def expiring?
     available? && auction.ended_at < (Time.current + 12.hours)
   end
 
+  def work_in_progress?
```

Comment:
I agree, the two fields are confusing and there feels like some overlap. I also like the idea of renaming `AuctionStatus` 

I do think that one challenge we have is that each status on the chain is not captured in `status` -- while we would be more state-machine-y, we are not forcing the statuses to go in a particular order so at least that is good.